### ISSUE TITLE
hangs when destroy-unattached is enabled

### ISSUE THREAD

**Issue by iloveitaly** [abhinav/tmux-fastcopy#160]
When I added

```
set -g destroy-unattached on
```

to my tmux config, tmux-fastcopy stopped working and just hunt indefinitely. Here's the verbose logs:

```
[tmux] display message: {message: #{pane_id}    #{window_id}    #{pane_width}  #{pane_height}   #{?#{pane_in_mode},#{pane_mode},normal-mode}    #{?#{==:#{pane_mode},copy-mode},#{scroll_position},0}   #{window_zoomed_flag}   #{pane_current_path}}
[tmux] show options: {global: true}
[tmux] new session: {command: [/Users/mike/.tmux/plugins/tmux-fastcopy/bin/tmux-fastcopy -pane %0 -action pbcopy -shift-action /Users/mike/.open-file-path.sh {} -regex path:(?:[\w\-\.]+|~)?(?:/[\w\-\.@]+){2,}(?:\:\d+)?(?:\:\d+)?\b -regex stacktrace-path:(?:[\w\-\.@/]+){2,}(?:\:\d+)(?:\:\d+)?\b -verbose -log /var/folders/xs/kzl6z1xs3cz8trwrgmmfhxbc0000gn/T/tmux-fastcopy1393702334 -tmux tmux], detached: true, env: [TMUX_FASTCOPY_WRAPPED_BY=3984], height: 25, width: 80}
[tmux] wait-for: TMUX_FASTCOPY_WRAPPER_3984

```

**Comment by abhinav** [comment]
Oh, that's icky. I can see why that would happen; tmux-fastcopy creates a temporary background session as an area to hold the pane while it's showing the window to select hints from.  That session is technically unattached, so it would be destroyed.

Looking around at the tmux help, there doesn't appear to be a to disable this option for a single session—might be possible for the session to set it for itself immediately upon creation but I suspect that'll race against tmux's destroy-unnatched.

I think what tmux-fastcopy will probably want to do here is check if the option is enabled at startup, and if so, instead of using a pane in a background session, use a pane in the same session. I'll try to look into this when I get a chance.

**Comment by iloveitaly** [comment]
Yeah, other option here is just throwing an error so the user is aware—there's other ways to kill unattached sessions if folks want.

**Comment by abhinav** [comment]
Yeah, that's fair. Depending on the complexity of the fix, I might error or make it opt-in. I'll dig into this later.

**Comment by iloveitaly** [comment]
@abhinav curious if you've thought about this further—would love to hear your take here!

**Comment by abhinav** [comment]
Hey, @iloveitaly. Thanks for the bump.
I previously made some progress on fixing this but wasn't able to finish it up at that time.
I was able to give it a little time today. There's a fix up at #171.
I've verified manually and with an integration test that the fix works, but please feel free to try it out on your end as well.
I'll try to release it soon.
